### ISSUE TITLE
When I enter a file, two lines of statusline appear.

### ISSUE THREAD

**Issue by CrazyCatZhang** [abzcoding/lvim#143]
<img width="1762" alt="image" src="https://github.com/abzcoding/lvim/assets/67346540/90bc0fef-5951-4a74-9eb2-93483829e3db">

But when I open other windows, it returns to normal, for example, when I invoke `nvimtree` using `<leader>+e`.

<img width="1762" alt="image" src="https://github.com/abzcoding/lvim/assets/67346540/48892ed8-62bc-4c8f-9e2f-c034f797e14e">

btw, neovim and lunarvim are both nightly~

**Comment by abzcoding** [comment]
yep, it seemed like a lualine isssue -> https://github.com/vimpostor/vim-tpipeline/issues?q=is%3Aissue+is%3Aclosed+double

adding this fixed it for me,
https://github.com/abzcoding/lvim/blob/1d8c415d0464ca1991075d4f3f521cc31f1f4401/lua/user/custom_user.lua#L18-L21


not sure what else can be done

**Comment by CrazyCatZhang** [comment]
I tried, but it didn't work.
```lua
if vim.env.TMUX then
    vim.cmd [[ autocmd VimEnter,WinEnter,BufEnter,VimResized * setlocal laststatus=0 ]]
end
```

When I enter a file and use `verbose set laststatus`, it indicates that the last time it was set was by lualine.

**Comment by CrazyCatZhang** [comment]
When I use other events, such as `setlocal nonumber`, it takes effect.

**Comment by abzcoding** [comment]
> When I use other events, such as `setlocal nonumber`, it takes effect.

not sure how to help you mate, it works fine for me, just disable it
```lua
lvim.builtin.tmux_lualine = false 
```

**Comment by CrazyCatZhang** [comment]
bro, I think it's because lualine's default configuration overrides the autocmd I set myself. If I execute that command after lualine setup, it should work. Is there a place in LunarVim where I can ensure that lualine is definitely set up?

**Comment by abzcoding** [comment]
> bro, I think it's because lualine's default configuration overrides the autocmd I set myself. If I execute that command after lualine setup, it should work. Is there a place in LunarVim where I can ensure that lualine is definitely set up?

check lunarvim documentation

**Comment by CrazyCatZhang** [comment]
I tried 
```lua
  lvim.builtin.lualine.on_config_done = function ()
    vim.opt.laststatus=0
  end
```
but it still doesn't work

**Comment by CrazyCatZhang** [comment]
Hey, bro, after I set a transparent window, the background of my spinner looks like this. How can I set its background color?
<img width="346" alt="image" src="https://github.com/abzcoding/lvim/assets/67346540/75978a24-f871-4b06-8486-cdc468fb0d8e">

**Comment by abzcoding** [comment]
> Hey, bro, after I set a transparent window, the background of my spinner looks like this. How can I set its background color?

fixed via this commit: https://github.com/abzcoding/lvim/commit/6c6bdebc50aa0d2d607fd7a0585d82e75a9dc469

**Comment by CassandraCat** [comment]
hey,mate,hot wo hidden the underline?
<img width="460" alt="image" src="https://github.com/abzcoding/lvim/assets/67346540/b0f1b28f-1b28-46b8-8052-39f4688d7ddb">

**Comment by abzcoding** [comment]
> hey,mate,hot wo hidden the underline? 

https://github.com/abzcoding/lvim/blob/1e40cea48d80dfd3d3c044a06c0695213e961843/lua/user/indent_blankline.lua#L50-L52

set scope `enabled = false`

**Comment by CassandraCat** [comment]
> > hey,mate,hot wo hidden the underline?
> 
> https://github.com/abzcoding/lvim/blob/1e40cea48d80dfd3d3c044a06c0695213e961843/lua/user/indent_blankline.lua#L50-L52
> 
> set scope `enabled = false`

solved. thanks bro~
